app/lib/submission_builder/formatting_methods.rb

```diff
@@ -18,6 +18,12 @@ def date_type(date)
       date.strftime("%F")
     end
 
+    def date_no_dash(date)
```

Comment:
unused?

Reply:
This was because I was trying to add SP_DOB_DT and the [[Working Doc] State File Data by State](https://docs.google.com/spreadsheets/d/1gTwfhe84ycTAXSuagk0UlT7sq6_9dmdrzE0j9EHiras/edit#gid=90796644) says the format should be "mmddyyyy". But I couldn't get SP_DOB_DT to work. Happy to remove this, not attached